From f66d0986462f8d3fe321dc20f6bd9904fc1e6a99 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 9 Jun 2015 14:32:19 +1000
Subject: [PATCH] hwdb: add Logitech TrackMan Marble Wheel USB

DPI is guesswork, no specs found on the web and calculating DPIs on a
trackball is tedious.

(cherry picked from commit 5967bda0bc9849bceeda393e3cd2ab04f2f25a03)
---
 hwdb/70-mouse.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb/70-mouse.hwdb b/hwdb/70-mouse.hwdb
index 2784b941a1..cfe3579fcd 100644
--- a/hwdb/70-mouse.hwdb
+++ b/hwdb/70-mouse.hwdb
@@ -169,6 +169,8 @@ mouse:usb:v046dpc00e:name:Logitech USB-PS/2 Optical Mouse:
 mouse:usb:v046dpc01b:name:Logitech USB-PS/2 Optical Mouse:
 # Logitech USB-PS/2 M-BT58
 mouse:usb:v046dpc03e:name:Logitech USB-PS/2 Optical Mouse:
+# Logitech TrackMan Marble Wheel USB
+mouse:usb:v046dpc401:name:Logitech USB-PS/2 Trackball:
  MOUSE_DPI=400@125
 
 # Lenovo USB mouse model MO28UOL
